packages/gatsby-source-mongodb/src/gatsby-node.js

```diff
@@ -148,6 +156,91 @@ function createNodes(
       })
       resolve()
     })
+  }).then(() => {
+    const changeStream = collection.watch()
+
+    changeStream.on(
```

Comment:
This should be conditional. Only run in `develop` with mongodb 4.2 or greater, and you need a replica set.

Reply:
Will recheck heuristic in https://docs.mongodb.com/manual/changeStreams/